apps/server/src/modules/server/api/dto/config.response.ts

```diff
@@ -220,6 +220,9 @@ export class ConfigResponse {
 	@ApiProperty()
 	FEATURE_BOARD_READERS_CAN_EDIT_TOGGLE: boolean;
 
+	@ApiProperty()
+	FEATURE_REGISTRATION_ENABLED: boolean;
```

Comment:
Maybe be more explicit to enable a reader of the code to understand, that it is part of the new feature:

```suggestion
	FEATURE_EXTERNAL_PERSON_REGISTRATION_ENABLED: boolean;
```